.stylelintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": [
+    "../../../core/.stylelintrc.json",
```

Comment:
I happen to have a free-standing `common_design` repo and this caused a fatal error when spinning up. I think we can ignore that because the theme will almost always sit inside a Drupal installation.

Just wanted to note it in case we can make a mention in the docs or something, since up until now it was working independent of a drupal installation.

Reply:
when I say free-standing I mean it's in a place such as `~/src/common_design` outside of a drupal theme directory.

Reply:
I think in general it's smart to be re-using Drupal config so aside from mentioning in docs that the error could pop up when used outside Drupal, no change necessary to the code.

Reply:
ok, I moved into a drupal installation and got the same error, but the file WAS indeed found in the expected relative path. I ended up resolving it by running `npm i` inside Drupal core itself. That fixed it for me.

Reply:
Great, thanks for pointing out the non-Drupal perspective. I'll add these details to the readme.

Reply:
I'm always torn about these decisions. Using the Drupal one makes it handy, but if we are catering for non-Drupal as well, having config in the base theme makes it more likely to be useful elsewhere and reduces the documentation (arguably).

There's an [eslint config file](https://github.com/UN-OCHA/common_design/blob/main/.eslintrc) in the base theme, and when I went looking at how we arrived at the decision, I got back to [CD-184](https://humanitarian.atlassian.net/browse/CD-184) and https://github.com/UN-OCHA/common_design/pull/23

@orakili Any recent thoughts on this? For the Drupal 8/9 version of RW, would you opt to keep the eslint config in the theme, or use Drupal core eslint config?

Reply:
To follow-up: coding standards and linting are theoretically most effective when our whole team uses one standard across multiple projects. So given that Drupal is our most frequently-used tool, IMO it's smart to use it as a standard.

I'd also move to implement the result of this PR into HID, for instance. I might have to copy some Drupal files, or tweak how we include the linting config, but ultimately if that means anyone on our team can pop into HID and make a change without much friction, we have improved the maintainability of HID by adopting the standard provided by Drupal.